packages/components/src/internal/components/administration/ProjectManagementPage.tsx

```diff
@@ -48,13 +57,25 @@ export const ProjectManagementPage: FC = memo(() => {
     );
 
     return (
-        <BasePermissionsCheckPage
-            hasPermission={user.isAdmin}
-            renderButtons={renderButtons}
-            title="Projects"
-            user={user}
-        >
-            <GridPanelWithModel allowViewCustomization={false} ButtonsComponent={Buttons} queryConfig={queryConfig} />
-        </BasePermissionsCheckPage>
+        <>
+            {successMsg && (
```

Comment:
Why is this outside the `BasePermissionsCheckPage`? Is there a scenario where we want to render the page as you don't have permissions but there is a success banner? Seems like this can be folded in above `GridPanelWithModel`.